sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/SampleAliquotDao.java

```diff
@@ -0,0 +1,21 @@
+package uk.ac.bbsrc.tgac.miso.persistence;
+
+import java.util.List;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleAliquot;
+
+public interface SampleAliquotDao {
```

Comment:
We decided to remove DAOs for specific sample categories (applies to SampleStockDao as well)